docs/control-structures.rst

```diff
@@ -28,6 +28,9 @@ something like::
         }
     }
 
+...note::
+    A function cannot accept a multi-dimensional array as an input parameter. This functionality is possible if you enable the experimental ``ABIEncoderV2`` feature by adding ``pragma experimental ABIEncoderV2;`` to your smart contract.
```

Comment:
Strictly, you don't add the feature to your smart contract, but rather to the source file. Otherwise fine.

Reply:
Done